docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -1,4 +1,4 @@
-# Style Library Interoperability
+# Style interoperability
```

Comment:
Why drop "library" here? I'm not sure that the phrase "style interoperability" makes sense. (I would like to change the name of this page, but that's for another PR 😛)

```suggestion
# Style library interoperability
```

Reply:
Arf ok, I was trying to keep a shorter name. Let's revert, one change at a time.